src/providers/details/tabs/Credentials/components/utils/getAuthType.ts

```diff
@@ -0,0 +1,23 @@
+import { OpenstackAuthType } from 'src/providers/utils/constants';
+
+export const getAuthType = (
+  authType: string | undefined,
+  username: string | undefined,
+): OpenstackAuthType => {
+  switch (authType) {
+    case 'token':
```

Comment:
please use SecretFieldsId enum for case keys

Reply:
Here those const values refer to field values and not field names, so it will be wrong to use those consts.